simulation/revolve2/simulation/scene/geometry/textures/_map_type.py

```diff
@@ -9,6 +9,7 @@ class MapType(Enum):
     - "CUBE": Wraps the texture around a cube object.
     - "SKYBOX": Like "cube" but maps onto the inside of an object.
     """
+
     MAP2D = "2d"
     CUBE = "cube"
     SKYBOX = "skybox"
```

Comment:
I think these changes should fix the issue. Maybe @andresgr96 could you test it on your side?